### PR TITLE
TwG Setup UI - Publication in Review - Update style

### PR DESCRIPTION
**Pull request by nfmohit**
## Summary

Addresses issue:

- https://github.com/google/site-kit-wp/issues/5527#issuecomment-1193959044

## Relevant technical choices

This PR intends to update the TwG Setup UI - Publication in Review scenario to match with the latest iteration (iteration four) of the Figma design. The iteration four uses a button for the action, whereas the current implementation mistakenly uses a link following an old iteration (second iteration).

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js [hunk 1]
```diff
@@ -39,17 +39,17 @@ export default function SetupPublicationPendingVerification() {
 				'google-site-kit'
 			) }
 		>
-			<Link
+			<Button
+				className="mdc-button--surfaces"
```
Comment (aaemnnosttv) [inline assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js:43]: Let's use a new `inverse` prop for this instead, similar to how we have one for `danger`.

File: assets/sass/vendor/_mdc-button.scss [hunk 2]
```diff
@@ -89,6 +89,11 @@
 		}
 	}
 
+	&--surfaces {
```
Comment (aaemnnosttv) [inline assets/sass/vendor/_mdc-button.scss:92]: Let's call this `inverse` to match the design token instead.

File: assets/sass/vendor/_mdc-button.scss [hunk 3]
```diff
@@ -89,6 +89,11 @@
...
+
+		@include mdc-button-filled-accessible($c-surfaces-on-background);
```
Comment (aaemnnosttv) [inline assets/sass/vendor/_mdc-button.scss:94]: Let's use `$c-surfaces-inverse-surface` here – the value is actually the same but this better matches the token in the design 👍 

image

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @nfmohit ! This is almost there, just a few tweaks for consistency 👍

File: assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js (hunk 1 above)
Comment (nfmohit) [reply assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js]: Updated, thank you!

File: assets/sass/vendor/_mdc-button.scss (hunk 2 above)
Comment (nfmohit) [reply assets/sass/vendor/_mdc-button.scss]: Updated, thank you!

File: assets/sass/vendor/_mdc-button.scss (hunk 3 above)
Comment (nfmohit) [reply assets/sass/vendor/_mdc-button.scss]: Updated, thank you!

**Comment by nfmohit** [comment]
Thank you for the kind review @aaemnnosttv! I've addressed the suggestions and updated the PR.

File: assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js [hunk 4]
```diff
@@ -39,17 +39,17 @@ export default function SetupPublicationPendingVerification() {
...
+			<Button
 				href="https://publishercenter.google.com/"
-				external={ true }
+				target="_blank"
 				aria-label={ __(
 					'Check your status on Thank with Google Publisher Center',
 					'google-site-kit'
 				) }
-				hideExternalIndicator={ true }
+				inverse={ true }
```
Comment (aaemnnosttv) [inline assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js:49]: We prefer to use the shorthand for boolean `true` props, so let's do the same here (I'm surprised we don't have a lint rule for this 🤔)

```suggestion
				inverse
```

**Review by aaemnnosttv — COMMENTED** [review]
Great! Just one tiny thing I'll apply real quick

File: assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js (hunk 4 above)
Comment (nfmohit) [reply assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js]: Ah, one more preference noted and I'll keep it in mind for future cases. Thank you for going ahead and applying @aaemnnosttv!

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks @nfmohit !

**Pull request merged**